cmd/util/ledger/migrations/cadence_values_migration_test.go

```diff
@@ -0,0 +1,356 @@
+package migrations
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io"
+	"testing"
+
+	"github.com/rs/zerolog"
+
+	_ "github.com/glebarez/go-sqlite"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/onflow/cadence/runtime/common"
+	"github.com/onflow/cadence/runtime/interpreter"
+	"github.com/onflow/cadence/runtime/sema"
+	"github.com/onflow/flow-go/cmd/util/ledger/reporters"
+	"github.com/onflow/flow-go/cmd/util/ledger/util"
+)
+
+const snapshotPath = "test-data/cadence_values_migration/snapshot"
```

Comment:
It would be great if somehow recorded the FVM/Cadence version with which the snapshot was created. Maybe in the filename?